docs/cimpl6LanguageReference.md

```diff
@@ -913,74 +909,35 @@ ICD10CM#C7B04  "Secondary carcinoid tumors of peritoneum"
 
 Following the header, the Value Set file contains value set definitions. The order of definitions does not matter. Each definition follows a required sequence of declarations. Follow the links for further explanation of each item:
 
-1. [`ValueSet` Declaration](#ValueSet) (required)
+1. [`ValueSet`](#ValueSet) declaration (required)
 1. [`Description`](#description) (optional but highly recommended)
-1. [Code Declarations](#explicit-codes) (optional)
-1. [`Includes Codes` ](#implicit-codes) (optional)
```

Comment:
Declaration and Code Declarations are intended to be substituted - italics.

Reply:
No, they are keywords. They are not to be substituted.